zomes/agent_activity/integrity/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "agent_activity_integrity"
+version = "0.1.0"
+edition.workspace = true
+description = "A Holochain integrity zome for agent activity"
+publish = false
+
+[lib]
+name = "agent_activity_integrity"
+path = "src/lib.rs"
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+hdi = { workspace = true }
+holochain_serialized_bytes = { workspace = true }
+serde = { workspace = true }
+
+# TODO watiing for a fix in 0.2.7-rc.1+
```

Comment:
Typo in `waiting`. This won't be a problem for much longer, 0.6 of Holochain upgrades to 0.3 of this library.